README.md

```diff
@@ -66,45 +66,78 @@ You can find more detailed installation instructions in [installing.md](docs/get
 
 ```python
 from concrete import fhe
+import numpy as np
+
 
 def add(x, y):
     return x + y
 
+
 compiler = fhe.Compiler(add, {"x": "encrypted", "y": "encrypted"})
-inputset = [(2, 3), (0, 0), (1, 6), (7, 7), (7, 1), (3, 2), (6, 1), (1, 7), (4, 5), (5, 4)]
```

Comment:
It was like this to show exactly what inputset should look like. When it's generated like we do ususally (like you did in the PR), I think it's harder to understand what it is. But it's okay for me if it's okay for @BourgerieQuentin as well.

Reply:
In my opinion, it's a matter of choice between (i) being easy to understand [your solution] (ii) being developer friendly [my solution], with easy-to-copy-paste code. So yes, I imagine there is no obvious choice.

Reply:
Could we keep the code and just add a comment to show the values (best of both worlds)?

something like:
```python
# Generate a random inputset looking like
# [(2, 3), (0, 0), (1, 6), (7, 7), (7, 1), (3, 2), (6, 1), (1, 7), (4, 5), (5, 4)]
inputset = [
    (
        np.random.randint(max_value_included + 1),
        np.random.randint(max_value_included + 1),
    )
    for _ in range(10)
]
```

Reply:
yes we could; @umut-sahin @BourgerieQuentin, what do you think?

Reply:
Looks good to me!

Reply:
Done. Letting this thread open for @BourgerieQuentin to approve